src/assets/style/helper.scss

```diff
@@ -0,0 +1,11 @@
+.flex {
+  &--center {
+    display: flex;
```

Comment:
consider introducing helper classes, to reduce [duplicate](https://github.com/ahr-wtag/react-todo-app/pull/1/files#diff-185c2133e77f93c7be3cfd69fed9832a7dd46afe313c43869f6b3a82b72cd9d3R8) styles.